apps/web/src/components/submission/index.tsx

```diff
@@ -53,8 +54,8 @@ const steps: StepType[] = [
 
 const handleValidate = (
   validator: (data: unknown) => Ret,
-  values?: SubmissionType,
-  key?: keyof SubmissionType,
+  values?: DeepPartial<SubmissionType>,
+  key?: keyof DeepPartial<SubmissionType>,
```

Comment:
What is the difference between a Partial and a DeepPartial?

Reply:
Almost the same thing as ValidateNested

`Partial<SubmissionType>` doesn't like when `personalInformation.firstName` can be undefined